pennylane/fermi/fermionic.py

```diff
@@ -324,6 +324,108 @@ def to_mat(self, n_orbitals=None, format="dense", buffer_size=None):
             wire_order=list(range(largest_order)), format=format, buffer_size=buffer_size
         )
 
+    def shift_operator(self, initial_position, final_position):
+        r"""Shifts an operator in the FermiWord from ``initial_position`` to ``final_position`` by applying the fermionic anti-commutation relations.
+
+            There are four anti-commutator relations:
+
+            .. math::
+
+                \begin{align*}
+                    a_0 a_1 &= -a_1 a_0\\
+                    a^{\dagger}_0 a^{\dagger}_1 &= -a^{\dagger}_1 a^{\dagger}_0\\
+                    a_0 a^{\dagger}_1 &= -a^{\dagger}_1 a_0\\
+                    a_0 a^{\dagger}_0 &= 1 - a^{\dagger}_0 a_0
+                \end{align*}
+
+
+            For more details see `Wikipedia <https://en.wikipedia.org/wiki/Creation_and_annihilation_operators#Creation_and_annihilation_operators_in_quantum_field_theories>`_.
```

Comment:
```suggestion
    There are four `anti-commutator relations <https://en.wikipedia.org/wiki/Creation_and_annihilation_operators#Creation_and_annihilation_operators_in_quantum_field_theories>`_:

    .. math::

        \begin{align*}
            a_0 a_1 &= -a_1 a_0\\
            a^{\dagger}_0 a^{\dagger}_1 &= -a^{\dagger}_1 a^{\dagger}_0\\
            a_0 a^{\dagger}_1 &= -a^{\dagger}_1 a_0\\
            a_0 a^{\dagger}_0 &= 1 - a^{\dagger}_0 a_0
        \end{align*}
```

Reply:
Corrected the indentation. Also, might use the following equations using curly brackets and and then define the format:

\left\{ a_i, a_j \right\} = 0
...

where :math:`\left\{ a_i, a_j \right\} = a_i a_j + a_j a_i`.